app/models/miq_report_result.rb

```diff
@@ -35,6 +35,17 @@ class MiqReportResult < ApplicationRecord
     end
   end
 
+  # These two hooks of a fix for the following BZ:
+  #
+  #   https://bugzilla.redhat.com/show_bug.cgi?id=1590908
+  #
+  # The remove extra data that is not necessary to be saved to the DB, but
+  # allow it to be retained for the remainder of the instanation incase the
+  # data is used to finish building the report results (the `after_commit` hook
+  # specifically, most likely this is overkill.
```

Comment:
the `after_commit` hook specifically, is probably overkill but allows us to not break existing code that expects the temporary chargeback data in the instantiated object

^ Maybe?